doc/tutorials/enable_s5.rst

```diff
@@ -153,9 +159,10 @@ Enable S5
       cd acrn-hypervisor
       make life_mngr
 
-#. For Service VM, LaaG VM and RT-Linux VM, run the Lifecycle Manager daemon:
+#. For the Service VM, LaaG VM, and RT-Linux VM, run the Lifecycle Manager
+   daemon:
 
-   a. Copy ``life_mngr.conf``, ``s5_trigger_linux.py``, ``user_vm_shutdown.py``, ``life_mngr``,
+   a. Copy ``life_mngr.conf``, ``s5_trigger_linux.py``, ``life_mngr``,
```

Comment:
These command examples assumes there's a network connection between the development and target.  We might mention that and say you can also use a USB stick to transfer files.